docs/framework/winforms/disable-dpi-awareness-visual-studio.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Disable DPI-awareness in Visual Studio
+ms.date: 08/14/2018
+ms.prod: visual-studio-dev15
+ms.technology: vs-ide-designers
+ms.topic: conceptual
+author: gewarren
+ms.author: gewarren
+manager: douge
+---
+# Disable DPI-awareness in Visual Studio
+
+Visual Studio is a dots per inch (DPI) aware application, which means displays scale automatically. If an application states that it's not DPI-aware, the operating system scales the application as a bitmap. This behavior is also called DPI virtualization. The application still thinks that it's running at 100% scaling, or 96 dpi.
+
+## Rendering issue
+
+The **Windows Forms Designer** in Visual Studio doesn't currently have scaling support. This can cause display issues when you open forms in the **Windows Forms Designer** on HDPI monitors.
+
+For example, the first image that follows shows **Windows Forms Designer** on a monitor at 100% scaling. The second image shows what the form looks like when it's opened on a high dots per inch (HDPI) monitor:
+
+![Windows Forms Designer scaling at 100%](media/scaling-100-percent.png)
+
+![Windows Forms Designer on HDPI monitor](media/win-forms-designer-hdpi.png)
+
+In Visual Studio 2017 version 15.8 and later, when you open a form in the **Windows Forms Designer**ga . on an HDPI monitor, Visual Studio displays a yellow message bar at the top of the designer:
+
+![Scaling on your monitor yellow message bar in Visual Studio](media/scaling-gold-bar.png)
+
+The message reads **Scaling on your main display is set to 200%. This might cause rendering problems in the designer window.**
```

Comment:
I think if we want to be precise, we should quote the message, which would be "Scaling on your main display is set to 200% (192 dpi). This might cause rendering problems in the designer window. How do I set scaling to 100%? Restart Visual Studio as a DPI-unaware process". But we might need to update this message anyway, then I'll update the doc as well.

Reply:
Since we are considering changing the message, we can paraphrase it here instead of trying to give an exact quote.

Reply:
I thought it would be good to include the message so customers can search on it. So if/when the message changes, if you can update the doc that would be great.